index.html

```diff
@@ -2,63 +2,26 @@
 <html>
 
   <head>
-    <style>
-      html {
-      font-family: sans-serif;
-      -webkit-text-size-adjust: 100%;
-      -ms-text-size-adjust: 100%;
-      }
-
-      body {
-      margin: 0;
-      }
-
-      .btn {
-      display: inline-block;
-      padding: 6px 12px;
-      margin-bottom: 0;
-      font-size: 14px;
-      font-weight: normal;
-      line-height: 1.42857143;
-      text-align: center;
-      white-space: nowrap;
-      vertical-align: middle;
-      -ms-touch-action: manipulation;
-      touch-action: manipulation;
-      cursor: pointer;
-      -webkit-user-select: none;
-      -moz-user-select: none;
-      -ms-user-select: none;
-      user-select: none;
-      background-image: none;
-      border: 1px solid transparent;
-      color: white;
-      text-decoration: none;
-      background-color: #aa0000;
-      margin: 18px 9px;
-      -o-border-radius: 7px;
-      -webkit-border-radius: 7px;
-      -moz-border-radius: 7px;
-      border-radius: 7px;
-      box-shadow: 0 1px 3px rgba(0, 0, 0, .35);
-      -webkit-box-shadow: 0 1px 3px rgba(0, 0, 0, .35);
-      -moz-box-shadow: 0 1px 3px rgba(0, 0, 0, .35);
-      -o-box-shadow: 0 1px 3px rgba(0, 0, 0, .35);
-      }
-    </style>
+    <link 
+      rel = "stylesheet"
+      type = "text/css"
+      href = "./stylesheet.css" 
+    />
   </head>
   
-  <body onload="makeMondrianImg();">
+  <body onload="makeArtistImg('Mondrian');">
```

Comment:
You can test the script's ability to imitate Earle Brown scores by changing the function's parameter value here from `"Mondrian"` to `"Brown"`.